app/scss/_utilities.scss

```diff
@@ -225,7 +225,7 @@ Background color with progressively-enhanced blur effect. Example:
 // Fluid text mixin courtesy of: https://css-tricks.com/snippets/css/fluid-typography/
 
 @function strip-unit($value) {
-  @return $value / ($value * 0 + 1);
+  @return calc(#{$value} / (#{$value} * 0 + 1));
```

Comment:
Please add comment on why calc is needed. Where is it defined?

Reply:
This was one of the deprecation warnings. The previous version of the code uses division outside of a calc function, and will not be supported in a future version of the Sass library that we use for JSchol. [Calc is a built-in function in Sass](https://sass-lang.com/documentation/values/calculations/).

Reply:
here's a [message about the deprecation warnings](https://stackoverflow.com/questions/71262427/deprecation-warning-using-for-division-outside-of-calc-is-deprecated-and-wi), we had a wall of them every time we would build or deploy JSchol.